release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -97,9 +96,52 @@ type of private DNS zone.
 
 // For more information on this new command and migrating Azure private DNS
 // zones, see ../installing/installing_azure/migrating-azure-dns-zones.adoc#migrating-azure-dns-zones[Migrating legacy Azure DNS zones].
-////
-// Monitor whether this info moves to a KB article or remains in official docs.
-// If it moves to KB, permanently remove this entry.
+
+[id="ocp-4-5-built-in-help-for-installconfig-supported-fields"]
+==== Built-in help for InstallConfig supported fields
+
+There is a new `openshift-install explain` command available that lists all the
+fields for supported InstallConfig versions including a short description
+explaining each resource. It also provides details on which fields are mandatory
+and specifies their default value. Using the `explain` command reduces the need
+to continually look up configuration options when creating or customizing the
+InstallConfig.
+
+[id="ocp-4-5-encrypt-ebs-instance-volumes-with-kms-key"]
+==== Encrypt EBS instance volumes with a KMS key
+
+You can now define a KMS key to encrypt EBS instance volumes. This is useful if
+you have explicit compliance and security guidelines when deploying to AWS. The
+KMS key can be configured in the `install-config.yaml` file by setting the
+optional `kmsKeyARN` field. For example:
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      rootVolume:
+        kmsKeyARN: arn:aws:kms:us-east-2:563456982459:key/4f5265b4-16f7-xxxx-xxxx-xxxxxxxxxxxx
+...
+----
+
+If no key is specified, the account's default KMS key for that particular region
```

Comment:
Suggest "If no key is specified, then the..."

Reply:
A previous mentor of mine drilled this saying into my head:
>If/then statements are for programming, not the English language.

It's like a subconscious voice in my head now, so I usually avoid it.

The ISG also appears to agree:
>if...then conj
Avoid using “then” to introduce an independent clause that follows an “if” clause. For example, write “If
you set a password, access is restricted,” not “If you set a password, then access is restricted.”
Use the “if...then” construction only to clarify that a conditional relationship exists between two actions or
events. For example, the “if...then” construction can improve clarity when you are joining two long clauses.

With that, I'm leaning towards keeping it the same. What do you think?

Reply:
Makes sense to me! I'll have to remember this myself going forward.